mbq/client/__version__.py

```diff
@@ -4,4 +4,4 @@
 __license__ = 'Apache 2.0'
 __title__ = 'mbq.client'
 __url__ = 'https://github.com/managedbyq/mbq.client'
-__version__ = '1.2.5'
+__version__ = '1.2.6'
```

Comment:
I apologize for my pedantry, but I think this breaking API change merits a major version bump.